src/nvim/api/vim.c

```diff
@@ -2451,3 +2451,70 @@ void nvim_del_user_command(String name, Error *err)
 {
   nvim_buf_del_user_command(-1, name, err);
 }
+
+/// Parse command line.
+///
+/// Doesn't check the validity of command or command arguments.
```

Comment:
This does fail for an non-existent command.

Reply:
Fixed now